src/vs/workbench/electron-browser/workbench.ts

```diff
@@ -493,6 +511,8 @@ export class Workbench extends Disposable implements IPartService {
 			}
 		}
 
+		this.menubarPart.updateStyles();
```

Comment:
@sbatten I would let the menu bar part listen on the fullscreen change itself to `updateStyles`.